media/videostitcher/stitcher_test.go

```diff
@@ -402,14 +410,98 @@ func TestCDNKeys(t *testing.T) {
 			r.Errorf("deleteCDNKey got\n----\n%v\n----\nWant to contain:\n----\n%v\n----\n", got, deleteCDNKeyResponse)
 		}
 	})
+	buf.Reset()
+}
+
+// TestLiveConfigs tests major operations on live configs. Create, list,
+// and get operations check if the live config resource name is returned. The
+// delete operation checks for a hard-coded string response.
+func TestLiveConfigs(t *testing.T) {
```

Comment:
Move tests for new samples into separate files (and a different PR).

See here:
https://googlecloudplatform.github.io/samples-style-guide/#dedicated-testing-per-sample